packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -93,6 +93,84 @@ registerStyles('vaadin-form-layout', formLayoutStyles, { moduleId: 'vaadin-form-
  * </vaadin-form-layout>
  * ```
  *
+ * ### Auto Responsive Mode
+ *
+ * To avoid manually dealing with responsive breakpoints, FormLayout provides an auto-responsive mode
+ * that automatically creates and adjusts fixed-width columns based on the container's available space.
+ * The {@link columnWidth} and {@link maxColumns} properties can be used to configure the desired column
```

Comment:
The `@link` syntax is not supported by Polymer Analyzer and is rendered as plain text.
Consider using the following instead to have proper links to the properties block:

```
[`columnWidth`](#/elements/vaadin-form-layout#property-columnWidth)
```

Not sure if `@link` would work for `.d.ts`, but we typically use the above syntax there as well.

Reply:
Thanks, fixed.